packages/phone-number-privacy/combiner/src/legacy/endpoints/signing/get-threshold-signature.ts

```diff
@@ -93,7 +92,15 @@ async function requestSignatures(request: Request, response: Response) {
   })
   obs.observe({ entryTypes: ['measure'], buffered: true })
 
-  request.headers[KEY_VERSION_HEADER] = _config.keys.version.toString()
+  const keyVersion = getRequestKeyVersion(request, logger) ?? _config.keys.currentVersion
+  request.headers[KEY_VERSION_HEADER] = keyVersion.toString()
+  const threshold = _config.keys.thresholds[keyVersion - 1]
```

Comment:
get-threshold-signature.ts is dead code we should delete eventually. It was frankly overkill for me to even put these changes in here since there isn't much point in maintaining it anymore. I do think that creating a helper function to clean this up is not worth the keystrokes at this point, though I agree it would be better in principle

Reply:
will delete this code

Reply:
- chatted sync: file will be deleted anyways